lib/main_micro_alp.pm

```diff
@@ -183,7 +183,7 @@ sub load_common_tests {
     loadtest 'microos/cockpit_service' unless is_staging || (is_microos('Tumbleweed') && get_var('HDD_1') =~ /-old/);
     # Staging has no access to repos and the MicroOS-DVD does not contain ansible
     # Ansible test needs Packagehub in SLE and it can't be enabled in SLEM
-    loadtest 'console/ansible' unless (is_staging || is_sle_micro || is_leap_micro);
+    loadtest 'console/ansible' unless (is_staging || is_sle_micro || is_leap_micro || is_alp);
```

Comment:
so, package is not available: https://openqa.opensuse.org/tests/3183757#step/ansible/36

Reply:
ansible will be in a container...

Reply:
Oh great, this means we will need a multi-machine job for running ansible on ALP. But I'm sure there will be an ansible package in ALP-Bedrock, no?

Reply:
> Oh great, this means we will need a multi-machine job for running ansible on ALP. But I'm sure there will be an ansible package in ALP-Bedrock, no?

I don't think so... and... why multimachine?

Reply:
> why multimachine?

Because unless you want to use the ansible container, you need another host where ansible is running, and which connects to the ALP host and applies the changes there.

Reply:
> > why multimachine?
> 
> Because unless you want to use the ansible container, you need another host where ansible is running, and which connects to the ALP host and applies the changes there.

yes, but then you are not testing the "ansible under test" :)